studio/src/app/popovers/editor/app-create-slide/app-create-slide.tsx

```diff
@@ -198,12 +198,18 @@ export class AppCreateSlide {
                         <p slot="content">Content</p>
                     </deckgo-slide-content>
                 </div>
-                <div class="item" custom-tappable onClick={() => this.addSlide(SlideTemplate.SPLIT)}>
+                <div class="item" custom-tappable onClick={() => this.addSlide(SlideTemplate.SPLIT_HORIZONTALLY)}>
                     <deckgo-slide-split class="showcase">
                         <p slot="start">Content</p>
                         <p slot="end">Content</p>
                     </deckgo-slide-split>
                 </div>
+                <div class="item" custom-tappable onClick={() => this.addSlide(SlideTemplate.SPLIT_VERTICALLY)}>
```

Comment:
Here we use `SlideTemplate.SPLIT` and create a specific method 

````
private async addSlide(template: SlideTemplate, deck?: Deck, attributes: SlideAttributes = undefined) {
``

then we extend `create-slides.utils.tsx` method `createSlideSplit()` with the attributes too

for inspiration, you could have a look to method `createSlideChart` in same class

Reply:
There already exists a method `addSlide`, should I just add the extra parameter?

Reply:
no need I think for the moment as it won't be use. we could always do that later

Reply:
Alright, I'll make a new method then.

Reply:
Coolio, thx